ci-automation/ci-config.env

```
@@ -45,3 +45,9 @@ QEMU_BIOS="/usr/share/qemu/bios-256k.bin"
 # Published by vms.sh as part of the qemu vendor build.
 QEMU_UEFI_BIOS="flatcar_production_qemu_uefi_efi_code.fd"
 
+
+# -- Equinix Metal --
+
+EQUINIXMETAL_PARALLEL="${QEMU_PARALLEL}"
+# Metro is a set of Equinix Metal regions
+EQUINIXMETAL_METRO="DA"
```

```suggestion
EQUINIXMETAL_METRO="SV"
```
to be near the bincache host